src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -245,7 +245,7 @@ using namespace Microsoft::Console::Render;
     DWRITE_MEASURING_MODE measuringMode,
     const DWRITE_GLYPH_RUN* glyphRun,
     const DWRITE_GLYPH_RUN_DESCRIPTION* glyphRunDescription,
-    IUnknown* /*clientDrawingEffect*/)
+    IUnknown* clientDrawingEffect)

```

Comment:
If a person wanted this effect disabled (for some unknown reason), we could pass in null here, right?

Reply:
Yeah you can pass in null. It checks null before attempting to QI it. Null means no effect.